action.yml

```diff
@@ -50,6 +50,10 @@ inputs:
   ISSUE_TITLE:
     description: "Title for issue"
     required: false
+  CREATE_ISSUE:
```

Comment:
Thanks for contributing! I agree It's a good idea to let user choose execution outcome. So, I would propose `EXECUTION_OUTCOME` input. It might accept `output`, `new-issue` or both values separated by comma. By default it will be `new-issue`. In your case it will be `output`. You can use `getMultipleValuesInput` to create array from input values, so you could use `forEach` method to handle all possible values.  I've been thinking about different results(new issue, artifact, comments), so it might have more variants in the future.

Reply:
I have implemented the `EXECUTION_OUTCOME` feature and would appreciate your review.

There's one concern I have. If all the values specified for `EXECUTION_OUTCOME` are invalid, the action ends silently without producing any output. I thought it would be appropriate to trigger an error in this scenario, but I'm not certain about the error handling policy.

Reply:
I think inputs should be validated before requests to avoid wasting rate limit quota on useless operations. This serves as input validation. If inputs are invalid and the operation is not feasible, you could just throw an error with an appropriate message. If some parameters are invalid but the operation is still useful, you could display a warning in the console.

Reply:
```typescript
if (
    ((!process.env.GITHUB_REPO_FOR_ISSUE ||
      !process.env.GITHUB_OWNER_FOR_ISSUE) &&
      (!core.getInput("GITHUB_OWNER_FOR_ISSUE") ||
        !core.getInput("GITHUB_REPO_FOR_ISSUE"))) ||
    (!core.getInput("GITHUB_OWNERS_REPOS") &&
      !process.env.GITHUB_OWNERS_REPOS) ||
    (!core.getInput("GITHUB_TOKEN") && !process.env.GITHUB_TOKEN)
  ) {
    throw new Error("Missing required variables");
  }
``` 
I think this should in a validate function with other validations. Probably messages should be more specific. Handling invalid input is in my to do list. I think I will work on it soon.